test/create-run-publish-app.ps1

```diff
@@ -0,0 +1,35 @@
+[cmdletbinding(PositionalBinding = $true)]
```

Comment:
Have you considered using Powershell scripts for both Debian and NanoServer?  The benefit is that there are one set of scripts to maintain as well as you can use the Powershell scripts to test Debian from a Windows machine.